### PR TITLE
Feature Admin Roles Allowed Actions

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Restrict the actions allowed for admin users
## Impacts

Admin GUI Admin Role
## NEWS file entries

New Features
++++++++++++
- Restrict the actions allowed for admin users
## Delete branch after merge

NO

### CODE REVIEW

**Comment by extrafu** [comment]
Is it meant to fix #1474 ?

**Comment by jrouzierinverse** [comment]
No this just restricts the actions that can be be selected for a user.

**Comment by louismunro** [comment]
Please rebase.

**Comment by jrouzierinverse** [comment]
Done

File: html/pfappserver/lib/pfappserver/Base/Form/Authentication/Action.pm [hunk 1]
```diff
@@ -128,8 +128,11 @@ sub options_actions {
         $self->form->ctx->log->error($@);
     }
     else {
-        $actions_ref = $classname->available_actions();
-        @actions = map { { value => $_, label => $self->_localize($_), attributes => { 'data-rule-class' => pf::Authentication::Action->getRuleClassForAction($_) } } } @{$actions_ref};
+        my @allowed_actions = $self->form->_get_allowed_options('allowed_actions');
+        unless (@allowed_actions) {
+            @allowed_actions = @{$classname->available_actions()};
+        }
+        @actions = map { { value => $_, label => $self->_localize($_), attributes => { 'data-rule-class' => pf::Authentication::Action->getRuleClassForAction($_) } } } @allowed_actions;
```
Comment (louismunro) [inline html/pfappserver/lib/pfappserver/Base/Form/Authentication/Action.pm:135]: Please break this on more than one line.
💅

**Pull request merged**